test/infamy/netns.py

```diff
@@ -272,3 +360,39 @@ def tcpdump(self, args=""):
                                  stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
                                  text=True, check=True)
         return tcpdump.stdout
+
+class TPMR(IsolatedMacVlans):
+    def __init__(self, a, b):
```

Comment:
Would've been nice to have a class description here as well, just like you did for IsolatedMacVlan.